.github/workflows/leak.yml

```
@@ -0,0 +1,42 @@
+name: Test compiler and tools for memory leaks
+
+on:
+  # This is for passing required checks
+  pull_request:
+  merge_group:
+
+jobs:
+  leaktest:
+    # Skip this for PRs
+    #
+    # FIXME: uncomment this before merge
+    # if: github.event_name != 'pull_request'
```

```suggestion
    if: github.event_name != 'pull_request'
```

Leaving this as a suggestion so it's easier to remember.